ui/apps/dashboard/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```

Comment:
Do we need to remove the `vite.svg` from code base?

Reply:
yes ~